test/test_cpp_extensions.py

```diff
@@ -13,6 +14,7 @@
 
 from torch.utils.cpp_extension import CUDA_HOME
 TEST_CUDA = torch.cuda.is_available() and CUDA_HOME is not None
+TEST_CUDNN = TEST_CUDA and torch._C.has_cudnn
```

Comment:
We should probably add `torch.cuda.cudnn_is_available()` to PyTorch

Reply:
I'm adding it here https://github.com/pytorch/pytorch/pull/8703

Reply:
much friendlier. good idea!